### ISSUE TITLE
List of Sites

### ISSUE THREAD

**Issue by dkullmann** [cyx/cuba-contrib#11]
@cyx did you remove the list of sites? The readme links to [cuba.is](cuba.is)

If you have a list of sites let's add some of the consulting work we've done from CB.

**Comment by dkullmann** [comment]
FYI I'm referring to the list mentioned [here](https://github.com/cyx/cuba-contrib#cuba-contrib)

**Comment by cyx** [comment]
Yeah not sure my memory is failing me. Maybe a list of companies using it
is the next version of it though.

**Comment by soveran** [comment]
@dkullmann I'm planning to add a list to cuba.is

**Comment by dkullmann** [comment]
@soveran anything I/we can do to help?

Charly and I can collaborate with you on how to add the list of sites to the UI (aka, do the design.) The biggest projects using Cuba from CB are probably:
- AT&T M2X — Enterprise grade connected devices platform
- Flat World Knowledge — Venture backed edtech, $10M in funding
- GoodUI Improve — One page app to comment on and improve UI's (from goodui.org)
- API Mission Control — White-label enterprise API management system
- Citrusbyte.com \o/

**Comment by soveran** [comment]
@dkullmann maybe a list of the sites with their URLs and logos?
